blog/2018-01-09-smart-testing-for-fast-feedback.textile

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Smart Testing For Fast Feedback
+author: hemanik
+tags: [ testing, guide, smart-testing ]
+---
+
+h1. Smart Testing For Fast Feedback
+
+p<>. ??As an engineer, you should constantly work to make your feedback loops shorter in time and/or wider in scope.??
+
+p>. ??"-@KentBeck":https://twitter.com/KentBeck/status/531964254946328576??
+
+
+One of the biggest issues that plague the software development life cycle is the slow feedback loop, the time between changing code (or tests) and getting results from the tests.
+
+h2. Impact of Slow Feedback Loop
```

Comment:
This whole section is very long. I would split it into smaller parts - subsections for example: "delays", "context switching", "resources"

Reply:
also, the name of this section could be just: "Motivation"